package.json

```diff
@@ -39,7 +45,24 @@
       "plugin:@typescript-eslint/eslint-recommended",
       "plugin:@typescript-eslint/recommended",
       "prettier",
+      "plugin:prettier/recommended",
       "prettier/@typescript-eslint"
-    ]
+    ],
+    "rules": {
```

Comment:
I want to go ahead and extract these to an internal (and can be published easily if we so desire) ruleset that is distinct from the `package.json` bits.